Ports/lxt/package.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=lxt
+version=1.3b
+files="http://stahlke.org/dan/lxt/lxt-${version}.tar.gz lxt-${version}.tar.gz 7a3ab299a6d61a71b271fd13b847b5a1c22a5f95df78561a325c78d50b6a6bc7"
+useconfigure='true'
+auth_type="sha256"
+depends=('ncurses' 'bash')
```

Comment:
Having one dependency per line would be nice.

Reply:
resolved locally, push coming soon